src/components/FormLogin/index.tsx

```diff
@@ -15,42 +21,83 @@ export const formLoginTestIds = {
   submit: 'form-login__submit',
 };
 
+interface FormInput {
+  email: string;
+  password: string;
+}
+
+const validateSchema = yup
+  .object({
+    email: yup.string().email().required(),
+    password: yup.string().required(),
+  })
+  .required();
+
 const FormLogin = () => {
+  const {
+    handleSubmit,
+    register,
+    formState: { errors },
+  } = useForm<FormInput>({
+    resolver: yupResolver(validateSchema),
+  });
+
+  const onSubmit: SubmitHandler<FormInput> = (data) => {
+    console.log(data);
+  };
+
   return (
-    <SimpleGrid spacing="6" data-test-id={formLoginTestIds.base}>
-      <FormControl>
-        <FormLabel htmlFor="email" data-test-id={formLoginTestIds.labelEmail}>
-          Email
-        </FormLabel>
-        <Input
-          id="email"
-          type="email"
-          data-test-id={formLoginTestIds.inputEmail}
-        />
-      </FormControl>
-      <FormControl>
-        <FormLabel
-          htmlFor="password"
-          data-test-id={formLoginTestIds.labelPassword}
-        >
-          Password
-        </FormLabel>
-        <Input
-          id="password"
-          type="password"
-          data-test-id={formLoginTestIds.inputPassword}
-        />
-      </FormControl>
-      <FormControl>
-        <Button
-          type="submit"
-          width="full"
-          data-test-id={formLoginTestIds.submit}
-        >
-          Sign in
-        </Button>
-      </FormControl>
-    </SimpleGrid>
+    <>
+      <FlashToast
+        mt="6"
+        title="Error"
+        messageList={[
+          errors.email?.message || '',
+          errors.password?.message || '',
+        ]}
+      />
+
+      <form onSubmit={handleSubmit(onSubmit)}>
+        <SimpleGrid mt="8" spacing="6" data-test-id={formLoginTestIds.base}>
+          <FormControl>
+            <FormLabel
+              htmlFor="email"
+              data-test-id={formLoginTestIds.labelEmail}
+            >
+              Email
```

Comment:
Just a small question here, do you have any plan to work on localization?

Reply:
@carryall No plan for that 🖖

Reply:
No plan for that 🙌